Tests/MongoSwiftSyncTests/SpecTestRunner/SpecTest.swift

```diff
@@ -67,8 +58,8 @@ internal struct TestCommandStartedEvent: Decodable, Matchable {
                 return false
             }
         }
-        return self.commandName.matches(expected: expected.commandName)
-            && self.command.matches(expected: expected.command)
+
```

Comment:
was this just a redundant check? I assume the command name is always in the document already?

Reply:
Yeah, the command name was being derived from the command document, so checking twice wasn't necessary. Also, we can no longer derive the command name from the document because of the ordering issues from JSON, so `self.commandName` is now optional, with it only being non-nil when initializing a `TestCommandStartedEvent` from an actual `CommandStartedEvent`.